docs/timelines.md

```diff
@@ -0,0 +1,141 @@
+Preload             | Preload the LTP values of a playback over a number of seconds
+Marker              | Mark a point in the timecode for information
+Wait for go         | Pause the timeline until you press Go to resume (only on internal timecode)
+
+- **Wait for go** triggers are useful when you have a preset sequence followed by an unknown delay, for
+example a win/walk-up sequence for an awards ceremony when you don't know how long the winner
+will take to reach the stage. Shapes will continue to run while the timeline is paused.
+
+Tracks
+------------------
+
+The timeline is split into **Tracks** which can be used to organise triggers into different groups or sections. Each different trigger in the timeline gets its own row in the track. 
+- You can mute tracks to prevent them playing back using the \{Power\} button.
+- You can lock tracks to prevent accidental changes to sections you've finished using the \{Lock\} button.
+- Tracks can be collapsed so they take up less room on the screen, using the arrow button to the left of the track name.
+
+Selecting timecode source
+------------------
+
+You can select one of the four available timecode sources to control your timeline from the [Timeline Options](./timelines/timeline-options.md#timecode-tab).
+
+To set up and enable each of the timecode sources as Internal, MIDI, Clock or SMPTE use the \[Timecode\] softkey at the top level main menu.
+
+Timecode linking and local timecode controls
+------------------
+
+The \{Link\} button in the bottom left of the Timeline View window links or unlinks the timeline with the timecode source. Especially with external timecode, it's useful when recording and editing to be able to control the timecode yourself rather than having to continually ask the video guys to rewind the source. So when this button is set to unlinked, you can control the timeline playback using the buttons in the top left corner of the Timeline View window as you would with an audio player. 
+
+- If you are using Internal Timecode or Winamp as a timecode source then these buttons will also operate the local Timecode source while linked (and will play/pause Winamp if that is the timecode source).
+
+![Timeline transport controls](/docs/images/Timeline-Timecode-Transport.png)
+
+- \{ |< \} sets the live timecode back to zero.
+- \{ |> \} starts playing from the start timecode.
+- \{ > \} and \{ || \} are play / pause controls.
+- \{ Stop \} stops playing and sets the live timecode back to zero.
+- \{ Record \} enters live record mode but doesn't start timecode playback.
+
+When you are linked to a timecode (except Internal Timecode and Winamp), the first four buttons are replaced by the timecode source name and only the Record button is shown.
+
+![Timeline external timecode controls](/docs/images/Timeline-Timecode-ExternalTransport.png)
+
+If the timecode source is disabled in the **Timecode** menu, the time is shown in red.
```

Comment:
```suggestion
If the timecode is disabled in the **Timecode** menu, the time is shown in red.
```
Timecode sources are not individually disabled or enabled. There is one options that rules them all.